oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/endpoint/EndpointTraffic.java

```diff
@@ -48,6 +48,7 @@ public class EndpointTraffic extends Metrics {
 
     public static final String SERVICE_ID = "service_id";
     public static final String NAME = "endpoint_traffic_name";
+    public static final String LAST_PING_TIME_BUCKET = "last_ping";
```

Comment:
I think endpoint traffic declared no update.

Could you verify the data in database?

Reply:
I have checked in Elasticsearch storage, looks like the update successful.
![image](https://github.com/user-attachments/assets/0764b13b-22b2-4c61-9b1a-437557503e0f)

Reply:
Check here, 

MetricsExtension(supportDownSampling = false, supportUpdate = false)

We declared this metric doesn't support update. What happens?

Reply:
https://github.com/apache/skywalking/blob/29618d7f2e795f1822d46015c23edc3f6b0c66be/oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/worker/MetricsPersistentWorker.java#L252-L257

According to this, an existing metric(if no_update=true) should not re-entry the `prepareRequests` list, and end in database again.